docs/private_net_tutorial.rst

```diff
@@ -0,0 +1,165 @@
+Raiden on Private Network Tutorial
+##################################
+
+Introduction
+============
+
+This tutorial shows how to run Raiden on a private network, using the ``master`` branch (this is useful when you are working on a pull-request).  This tutorial assumes Ubuntu 18.04.2 LTS and ``bash``.
+
+Creating a Virtual Environment
```

Comment:
This is duplicated documentation, just point the user to the [installation page](https://raiden-network.readthedocs.io/en/stable/overview_and_guide.html#installation) otherwise we will have multiple places to keep up to date.

Reply:
This requirement conflicts with my want.  I want a self-contained guide that anybody can follow to and produce the same state.